server/odc-service/src/main/java/com/oceanbase/odc/service/permissionapply/project/ApplyProjectPreprocessor.java

```diff
@@ -66,6 +66,10 @@ public void process(CreateFlowInstanceReq req) {
         List<ResourceRoleEntity> resourceRoleEntities = checkResourceRoleExist(
                 parameter.getResourceRoles().stream().map(ApplyResourceRole::getId).collect(Collectors.toList()));
         parameter.setUserId(authenticationFacade.currentUserId());
+        parameter.getProject().setName(projectEntity.getName());
+        for (int index = 0; index < parameter.getResourceRoles().size(); index++) {
+            parameter.getResourceRoles().get(index).setName(resourceRoleEntities.get(index).getRoleName());
```

Comment:
Why do you need the project role name?

Reply:
The front-end should display it in task details. If we return it, the front-end will no need extra query.